test/stubs/contentful/get-entries-campaign-with-overrides.json

```diff
@@ -0,0 +1,40 @@
+{
+  "sys": {
+    "type": "Array"
+  },
+  "total": 1,
+  "skip": 0,
+  "limit": 100,
+  "items": [
+    {
+      "sys": {
+        "space": {
+          "sys": {
+            "type": "Link",
+            "linkType": "Space",
+            "id": "owik07lyerdj"
+          }
+        },
+        "id": "6tlsrM2RTa40oMqwcIKSg6",
+        "type": "Entry",
+        "createdAt": "2017-02-15T20:30:31.426Z",
+        "updatedAt": "2017-05-12T15:20:14.634Z",
+        "revision": 10,
+        "contentType": {
+          "sys": {
+            "type": "Link",
+            "linkType": "ContentType",
+            "id": "campaign"
+          }
+        },
+        "locale": "en-US"
+      },
+      "fields": {
+        "campaignId": "7",
+        "gambitSignupMenuMessage": "Great! Little moments can have a big impact. Complimenting someone can turn their whole day around.\n\nDo this: post encouraging notes in places that can trigger low self-esteem, like school bathrooms.\n\nThen, text START to share a photo of the messages you posted (and you'll be entered to win a $1500 scholarship)! \n\nBonus: Text back a photo of your posted note by 11:59PM tonight and you can enter to win a gift card of your choice! (Winners announced this week). ",
```

Comment:
I understand your concern. Let me give you an example, lets say that this updater exists now, and I run this stub updater. The text that you referenced up there would not be affected by it, it would still say: `text START`, because it is not a product of using any variable to set it, it is actually hardcoded in contentful.

It seems like we are hardcoding this variables as part of creating overrides for certain campaign fields in contentful. I believe what we want is to replace that `text START` in contentful for `text {{cmd_reportback}}`. This way, the stub would have the template where `helpers.replacePhoenixCampaignVars` could work its wonderful magic and replace with the appropriate variable (whatever is in the env variable).

There is a possibility that , as we add more JSON mocks, we end up storing a few that do store "rendered" data, which depends on env variables. I think we should run away as much as we can from this pattern, since it introduces what your concern is. You can see how I made a "U" turn and stored the [all-campaign-fields.json](https://github.com/DoSomething/gambit/pull/903/commits/7fea8ed5dec16fbd21bb4e6bfd9a42e44c55a7cd#diff-8ec8b41db0fdf2647ef25e54948245ce) Mock, without the `.rendered` property and just tested that the actual function does populate this property with something, here: [renderAllMessagesForPhoenixCampaign should inject a rendered property to each campaign field](https://github.com/DoSomething/gambit/pull/903/commits/d8d369516fe57ec6164d57e0d73d4d31c23baa06#diff-64a8fd8e22ad28db222c591fde9911eeR383)